sympy/series/formal.py

```diff
@@ -942,6 +943,14 @@ def __new__(cls, *args):
         args = map(sympify, args)
         return Expr.__new__(cls, *args)
 
+    def __init__(self, *args):
+        ak = args[4][0]
+        k = ak.variables[0]
+        self.ak_seq = sequence(ak.formula, (k, 1, oo))
+        self.fact_seq = sequence(factorial(k), (k, 1, oo))
+        self.bell_coeff_seq = self.ak_seq * self.fact_seq
+        self.sign_seq = sequence((-1, 1), (k, 1, oo))
+
```

Comment:
Can the following be done,
```python
def __new__(cls, *args):
    args = map(sympify, args)
    obj = Expr.__new__(cls, *args)
    ak = args[4][0]
    k = ak.variables[0]
    obj.ak_seq = sequence(ak.formula, (k, 1, oo))
    obj.fact_seq = sequence(factorial(k), (k, 1, oo))
    obj.bell_coeff_seq = self.ak_seq * self.fact_seq
    obj.sign_seq = sequence((-1, 1), (k, 1, oo))
    return obj
```

Reply:
I initially proceeded with this, but was changed based on @leosartaj 's suggestions. I think it's better to define such variables in `__init__` because they are straight-away initializations.

Reply:
Okay, resolve this conversation.